### PR TITLE
Update elections information

### PR DESCRIPTION
**Pull request by 62442katieb**

### CODE REVIEW

File: content/elections/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Please self-nominate yourself [HERE](https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAAMAAC9pqdJUN0xITDMxUktDT0RCUjJMVlc1OTVSRTlTRS4u!)
+
+The open-science movement is one of the best things happening in neuroscience. Hackathons, open-science room, new tools and methods, reproducibility and reliability, open access and inclusivity: the OS SIG is the place where it’s all happening!
+
+If open-science excites you and you want to help push its frontiers, please consider applying for a position in the OS SIG! 
+
+[Early career researchers](https://www.ohbmtrainees.com/), BIPOC, LGBTQ+, and people from outside Europe and North America especially encouraged to apply! 
+
+All OS_SIG members have reduced or no-cost registration for OHBM 2021!
```
Comment (emdupre) [inline content/elections/index.md]:
```suggestion
All OS SIG members have reduced or no-cost registration for OHBM 2021!
```

Comment (62442katieb) [reply content/elections/index.md]: I like it! I think I'll change "members" to "officers", just to keep that delineation throughout.

**Review by jbpoline — APPROVED** [review]

**Comment by emdupre** [comment]
This PR isn't yet showing up on the netlify deploy (ossig.netlify.app) -- I'm not sure if you need to authorize something, @cmaumet, to get it on the main site ?

**Comment by cmaumet** [comment]
Thanks for the ping @emdupre! Yes, unfortunately we have to publish manually (as publish sometimes fail -- additional pages not loading -- unpredictably... cf. [here](https://github.com/ohbm/ossig#deploying)). Would be good to pass over those credentials, @62442katieb would you like to take this up?

**Comment by cmaumet** [comment]
Now online: https://ossig.netlify.app/elections/ (sorry for the delay)

**Comment by 62442katieb** [comment]
@cmaumet Sorry for the delay! Yes, I'm happy to take on the website mantle. What information do you need from me to make the transition?

**Comment by cmaumet** [comment]
@62442katieb: we can't do a "transfer" but I can deactivate on my end and you can activate on your end. You will need an account on netlify. Can you let me know when you have it ready?

**Pull request merged**